_posts/minutes/2016-03-18-minutes.markdown

```diff
@@ -1 +1,35 @@
+---
+---
+Begin: 01:05pm 
+### Introduction
+##A-Reports: 
+  -ICC: Spring Events:  
+  -Event Committee Report: Shirt Counts, Field Trip Sign-Up, Garden Fest & Extra Life Volunteer Sign-UpExtra Life Theme:         Shooter Games (Halo, CoD, GoW); Racing Games Tournament Room (Super Smash Bros) -Add on Info. For the events (Extra Life)   -Garden Fest: 9am-4pm 
```

Comment:
This looks like it is supposed to be a bulleted list?

`-Garden Fest: 9am-4pm` should probably be on its own line